sklearn/decomposition/online_lda.py

```diff
@@ -277,13 +282,18 @@ def __init__(self, n_topics=10, doc_topic_prior=None,
         self.n_jobs = n_jobs
         self.verbose = verbose
         self.random_state = random_state
+        self.n_topics = n_topics
 
     def _check_params(self):
         """Check model parameters."""
+        if self.n_topics is not None:
+            self.n_components = self.n_topics
```

Comment:
So I should add a private variable that gets assigned either the value of n_topics or n_components, instead of overwriting n_components, yes?